### PR TITLE
Updated themes 

### PR DESCRIPTION
**Pull request by tamojit-123**
Added two themes yellow theme and material dark theme.

### CODE REVIEW

**Comment by tamojit-123** [comment]
@ashutosh1919, please check

File: src/theme.js [hunk 1]
```diff
@@ -119,4 +119,28 @@ export const orangeTheme = {
...
+  highlight: "#4f5b62",
+  dark: "#000a12",
+  secondaryText: "#ffffff",
+  imageHighlight: "#607d8b",
+  compImgHighlight: "#E6E6E6",
+  jacketColor: "#607d8b",
+  headerColor: "#1E272C",
+};
+
 export const chosenTheme = blueTheme;
```
Comment (ashutosh1919) [inline src/theme.js:146]:
```suggestion
export const chosenTheme = yellowTheme;
```

**Comment by ashutosh1919** [comment]
Screenshot 2020-12-26 at 11 41 35 AM
@tamojit-123 , person clothes are not in contrast in yellow theme. Please look at the deploy preview. https://deploy-preview-98--ashutosh1919portfolio.netlify.app/#/home

**Comment by tamojit-123** [comment]
@ashutosh1919 I will look into, but what about the material dark theme.

**Comment by ashutosh1919** [comment]
Can you please fix the yellow theme ? Then once that is fixed, i will review that

**Pull request closed without merging**